### DISCUSSION TITLE
Notable uses of Roda

### DISCUSSION THREAD

**Discussion by teamschlaflos** [jeremyevans/roda#350] [General]
Are there any well-known companies or websites that have utilized the Roda framework in notable ways?

**Comment by jeremyevans** [comment]
There is a list of open source applications using Roda at the bottom of https://roda.jeremyevans.net/documentation.html.  I would guess most usage is commercial and not open source.

**Comment by jaredcwhite** [comment]
I'm way behind on publicizing this to a more mainstream Ruby audience, but I've been working a lot this month on version 2 of the [Bridgetown framework](https://www.bridgetownrb.com) which heavily relies on Roda, and soon Rodauth and Sequel, to provide all of the server-backed functionality. It's not a stretch to say every Bridgetown site is a Roda site in development, and of the ones that aren't strictly static sites in production, those too are using Roda. I've made real money at this point using these tools in my own business, so in my mind at least, the utility and value of this stack is truly notable!

One thing I'm doing as part of the v2 release cycle is starting to pull more pieces out of the Bridgetown + Roda integration (and stuff I've cobbled together on individual projects) that can be installed separately as plugins in a more idiomatic Roda way. I've also started investigating going the inverse direction from what we did in our integration to date, such that eventually you could take a stock Roda app and then swap out its own view rendering/assets/static files/etc. handling and switch on Bridgetown.

I'm just about to start documenting/publicizing all of this stuff in more detail at [Fullstack Ruby](https://www.fullstackruby.dev), as I think the Roda/Rodauth/Sequel combo is extremely rad but it can be a bit daunting without code examples & tutorials. Stay tuned! ☺️

**Comment by kennyfrc** [comment]
A notable company is [Ubicloud](https://www.ubicloud.com/). I'm guessing they have the largest open source Roda codebase: [Github repo](https://github.com/ubicloud/ubicloud/tree/main).

**Reply by jeremyevans** [reply]
Thanks, I added this in fd90c1deda4c1de428a04eaf9db11cc36ffe9593
